### DISCUSSION TITLE
anyone install Mirasvit Elastic Search Ultmate on breezefront ?

### DISCUSSION THREAD

**Discussion by ppeyman** [breezefront/community#317] [5. General]
Hello, has anyone installed Mirasvit Elastic Search Ultimate on a blank theme?
And is it compatible with breezefront?

**Comment by sandipansaha** [comment]
Hi @ppeyman 
Yes, we do.
You need to install the following packages -

mirasvit/module-search-ultimate:^2.1
swissup/breeze:^2.0
swissup/module-breeze-mirasvit-search-ultimate:^1.0.2

Thanks
